syntax = "proto2";
import "game_commands.proto";

message Command_ChangeZoneProperties {
    extend GameCommand {
        optional Command_ChangeZoneProperties ext = 1031;
    }
    optional string zone_name = 1;

    // Reveal top card to all players.
    optional bool always_reveal_top_card = 10;
    // reveal top card to the owner.
    optional bool always_look_at_top_card = 11;
}
